### ISSUE TITLE
token not revoked when logout. 

### ISSUE THREAD

**Issue by jdgong** [alexruperez/PFLinkedInUtils#15]
I am not sure if this is due to the some new changes to LinkedIn's API for iOS9. Now once I logged in on one device, I cannot log in with the same LinkedIn account on anther device, even if I have already logged out of the first device. It seems LikedIn still have the access token associated with the first device. Am I missing some thing that I need to call a specific function when logging out to revoke the token? Please help.

**Comment by alexruperez** [comment]
Did you check the new [App Transport Security](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/) for iOS 9?

**Comment by jdgong** [comment]
Yes, and I have no problem logging in on a iOS9 device. The issue is once I am logged in, I cannot log in anymore with other devices, even after I logged out on the first device using PFUser.logout(). PS, I can still log in to the first device afterwards.

**Comment by ivan-liljeqvist** [comment]
@jdgong I have the same problem! Have you solved it? I'm thinking having a separate LinkedInUser for each device to isolate the sessions. @alexruperez what do you think?

**Comment by jdgong** [comment]
@ivan-liljeqvist  I haven't resolved it yet and cannot login to LinkedIn in multiple devices. Hope @alexruperez can shed some light on it.

**Comment by ivan-liljeqvist** [comment]
I have a solution - if the login fails because of access toke missmatch use
    [PFLinkedInUtils clearUserDefaults];
    [PFLinkedInUtils logInWithBlock:block];

to reset the token on that particular device. The user will have to log in with LinkedIn again. It's not perfect because the user has to login again, but works.
